python/ray/serve/endpoint_state.py

```diff
@@ -108,6 +108,12 @@ def shadow_traffic(self, endpoint: EndpointTag, backend: BackendTag,
         self._checkpoint()
         self._notify_traffic_policies_changed(endpoint)
 
+    def get_endpoint_route(self, endpoint: EndpointTag) -> Optional[str]:
```

Comment:
Do we need to worry about variable routes here?  If there's some nontrivial interaction, maybe we should document it and add a test.

Reply:
Actually I think this is fine, we're just matching the endpoint tag, not the route itself